app/models/shipit/deploy_spec/npm_discovery.rb

```diff
@@ -23,18 +32,23 @@ def discover_yarn_checklist
         if yarn?
           [%(<strong>Don't forget version and tag before publishing!</strong> You can do this with:<br/>
             yarn version --new-version <strong>&lt;major|minor|patch&gt;</strong>
-            && git push --follow-tags</pre>)]
+            && git push --follow-tags</pre>), publish_config_checklist.first]
         end
       end
 
       def discover_npm_checklist
         if npm?
           [%(<strong>Don't forget version and tag before publishing!</strong> You can do this with:<br/>
             npm version <strong>&lt;major|minor|patch&gt;</strong>
-            && git push --follow-tags</pre>)]
+            && git push --follow-tags</pre>), publish_config_checklist.first]
         end
       end
 
+      def publish_config_checklist
+        [%(<strong>Don't forget publishConfig settings in your package.json!</strong> See
+          <a target="_blank" href="https://development.shopify.io/guides/gems/">docs</a>)]
```

Comment:
This link will need to be updated once I get the doc up

Reply:
Removed this completely in favour of the `misconfigured-npm-publish-config` script